new-lamassu-admin/src/pages/Commissions/helper.js

```diff
@@ -39,7 +39,12 @@ const cashOutHeader = (
 const getView = (data, code, compare) => it => {
   if (!data) return ''
 
-  return R.compose(R.prop(code), R.find(R.propEq(compare ?? 'code', it)))(data)
+  // The following boolean should come undefined if it is rendering an unpaired machine
+  const hasValue = R.find(R.propEq(compare ?? 'code', it))(data)
+
+  return hasValue
+    ? R.compose(R.prop(code), R.find(R.propEq(compare ?? 'code', it)))(data)
```

Comment:
Why not just do R.prop(code) of hasValue?

Reply:
You're correct, fixing